ramls/loan-policy.json

```diff
@@ -0,0 +1,173 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string"
+    },
+    "name": {
+      "type": "string"
+    },
+    "description": {
+      "type": "string"
+    },
+    "loanable": {
+      "type": "boolean"
+    },
+    "loansPolicy": {
+      "type": "object",
+      "additionalProperties": false,
```

Comment:
Should we allow additional properties in case of schema updating in mod-circulation?